index.html

```diff
@@ -229,5 +244,60 @@ <h2 class="spaced">About me</h2>
                 </div>
             </div>
         </section>
+
+        <footer>
+        <div class="contact-form">
+            <p class="p-contact">I'm always interested in hearing about 
+                new projects, so if you'd like to chat 
+                please get in.</p>
+            
+        <form id="contact-me" action="https://formspree.io/f/contact-me" method="post">
+            <input 
+            type="text" 
+            id="name" 
+            placeholder="Full name" 
+            maxlength="30" 
+            name="full name" required>
+            
+            <input 
+            type="email" 
+            id="email" 
+            placeholder="Email address" 
+            name="email" required>
+            
+            <textarea 
+            name="message" 
+            id="text" 
+            cols="30" rows="10" 
+            placeholder="Enter text here" 
+            maxlength="500" required></textarea>
+
+            <button type="submit" class="contact-btn">Get in touch</button>
```

Comment:
- [ ] [OPTIONAL]: You should consider adding some styles for the button's actions, such as pressed and hover. When I was testing the form, it was a bit hard to tell if the button was working or not.

Reply:
Thank you for the reviews I have effected these changes